plugins/netctl/cmdimpl/nodes.go

```diff
@@ -75,3 +78,43 @@ func PrintNodes() {
 
 	w.Flush()
 }
+
+// getNodeInfo will make an http request for the given command and return an indented slice of bytes.
+func getNodeInfo(client *remote.HTTPClient, base string, cmd string) ([]byte, error) {
+	res, err := client.Get(base, cmd)
+	if err != nil {
+		err := fmt.Errorf("getNodeInfo: url: %s Get Error: %s", cmd, err.Error())
+		fmt.Printf("http get error: %s ", err.Error())
+		return nil, err
+	} else if res.StatusCode < 200 || res.StatusCode > 299 {
+		err := fmt.Errorf("getNodeInfo: url: %s HTTP res.Status: %s", cmd, res.Status)
+		fmt.Printf("http get error: %s ", err.Error())
+		return nil, err
+	}
+
+	b, err := ioutil.ReadAll(res.Body)
+	if err != nil {
```

Comment:
defer res.Body.Close()

Reply:
Updated + added defer b.close() at other place where request is used